src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -3683,6 +3653,90 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 								}
 							}
 							break;
+						//
+						// GetConstructor (Type[])
+						// GetConstructor (BindingFlags, Binder, Type[], ParameterModifier [])
+						// GetConstructor (BindingFlags, Binder, CallingConventions, Type[], ParameterModifier [])
+						//
+						case "GetConstructor" when calledMethod.DeclaringType.Name == "Type"
+						  && calledMethod.Parameters.Count >= 1
+						  && calledMethod.DeclaringType.Namespace == "System": {
+
+								reflectionContext.AnalyzingPattern ();
+								var parameters = calledMethod.Parameters;
+								int? ctorParameterCount = null;
+								BindingFlags bindingFlags = BindingFlags.Default;
+								if (parameters.Count > 1) {
+									if (methodParams [1].AsConstInt () != null)
+										bindingFlags |= (BindingFlags)methodParams [1].AsConstInt ();
+								}
+								// Go over all types we've seen
+								foreach (var value in methodParams [0].UniqueValues ()) {
+									if (value is SystemTypeValue systemTypeValue) {
+										MarkMethodsFromReflectionCall (ref reflectionContext, systemTypeValue.TypeRepresented, ".ctor", bindingFlags, ctorParameterCount);
+									} else if (value == NullValue.Instance) {
```

Comment:
Vitek added a helper to handle the non-SystemTypeValue nodes. I see that the code here is a copy of how Activator.CreateInstance was handled before Vitek's change. Can you update this to how Activator.CreateInstance is handled now?